app/actions/remote/thread.ts

```diff
@@ -291,7 +291,7 @@ export const syncThreadsIfNeeded = async (serverUrl: string, isCRTEnabled: boole
 
         if (teams?.length) {
             for (const team of teams) {
-                promises.push(syncTeamThreads(serverUrl, team.id, true, true));
+                promises.push(syncTeamThreads(serverUrl, team.id, {excludeDirect: true, fetchOnly: true}));
```

Comment:
Just curious - where do we sync the DM/GM threads?

Reply:
In other calls to `syncTeamThreads`, where we don't pass the `excludeDirect` option. Mainly on startup / reconnect, if I am not mistaken.

Reply:
Exclude direct is false also for the initial team, this will fetch the threads on DM's and GM's that are returned for every team if you don't set the flag as true